docs/api-overview.mdx

```diff
@@ -115,7 +122,125 @@ The same ApiKey necessary to call /hasMatchingCredentials grants you and only yo
+      "issuers": [],
+      "mandatory": "yes"
+    },
+    {
+      "type": "AddressCredential",
+      "issuers": [],
+      "children": [
+        {
+          "type": "CityCredential",
+          "issuers": [],
+          "mandatory": "if_available"
+        },
+        {
+          "type": "StateCredential",
+          "issuers": [],
+          "mandatory": "yes"
+        },
+        {
+          "type": "CountryCredential",
+          "issuers": [],
+          "mandatory": "yes"
+        },
+        {
+          "type": "ZipCodeCredential",
+          "issuers": []
+        }
+      ]
+    }
+  ]
+}
+```
+
+</p>
+</details>
+
+In the given example, the user will be asked to share their `SsnCredential`, `FullNameCredential` and `AddressCredential` credentials.
+
+Let's breakdown to understand what is happening here:
+
+- `SsnCredential` is a "[single attribute atomic credential](/schema#schema-definitions)" and therefore does not have any sub-credentials. Because it is marked as `mandatory`, it only matches if the user has a credential.
+
+```json title="SnnCredential"
+{
+  "type": "SsnCredential",
+  "issuers": [],
+  "mandatory": "yes"
+}
+```
+
+- `FullNameCredential` is a "[composite credential](/schema#schema-definitions)" and therefore has *2 sub-credentials*¹: `FirstNameCredential`, `LastNameCredential`. Because it is marked as `mandatory`, and it has no `children` attribute, it only matches if the user has a credential with all sub-credentials.
+
+```json title="FullNameCredential"
+{
+  "type": "FullNameCredential",
+  "issuers": [],
+  "mandatory": "yes"
+}
+```
+
+> _¹The `FullNameCredential` also has a `MiddleNameCredential`, but it's `optional`._
+
+- The `AddressCredential` credential is made up of 6 sub-credentials: `Line1Credential`, `Line2Credential`, `CityCredential`, `StateCredential`, `CountryCredential` and `ZipCodeCredential`.
+
+  But because we specified a `children` attribute, we are only asking for the `CityCredential`, `StateCredential`, `CountryCredential` and `ZipCodeCredential` sub-credentials. So `Line1Credential` and `Line2Credential` are ignored.
+
+  ```json title="AddressCredential"
+  {
+    "type": "AddressCredential",
+    "issuers": [],
+    "children": [
+      {
+        "type": "CityCredential",
+        "issuers": [],
+        "mandatory": "if_available"
+      },
+      {
+        "type": "StateCredential",
+        "issuers": [],
+        "mandatory": "yes"
+      },
+      {
+        "type": "CountryCredential",
+        "issuers": [],
+        "mandatory": "yes"
+      },
+      {
+        "type": "ZipCodeCredential",
+        "issuers": []
+      }
+    ]
+  }
+  ```
+
+  The user will be asked to share the `CityCredential` if available (this mean if the user has a credential of type `CityCredential` it will mandory be shared, but if the user does not have a credential of type `CityCredential` the user will still be able to share the `AddressCredential` credential).
```

Comment:
```suggestion
The user will be asked to share the `CityCredential` if available (this means if the user has a credential of type `CityCredential` it will be mandatory, but if the user does not have a credential of type `CityCredential` the user will still be able to share the `AddressCredential` credential).
```